### PR TITLE
adding fonts

### PR DESCRIPTION
**Pull request by YuliaSimanavets**

### CODE REVIEW

File: RingSizer/Base/Extensions/Fonts.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//  FirstMVVM
+//
+//  Created by Yuliya on 04/12/2023.
+//
+
+import Foundation
+import UIKit
+
+extension UIFont {
+    static func roundedRegular(size: CGFloat) -> UIFont {
```
Comment (vkostechko-applace) [inline RingSizer/Base/Extensions/Fonts.swift:12]: please update swift gen config and use here generated enum

**Review by vkostechko-applace — CHANGES_REQUESTED** [review]

File: RingSizer/Base/Extensions/Fonts.swift (hunk 1 above)
Comment (YuliaSimanavets) [reply RingSizer/Base/Extensions/Fonts.swift]: got it

File: RingSizer/Base/Extensions/Fonts.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        return UIFont(name: "SFProRounded-Regular", size: size) ?? UIFont.systemFont(ofSize: size)
+    }
+    static func roundedMedium(size: CGFloat) -> UIFont {
```
[the remark this answers is not in the document]
Comment (YuliaSimanavets) [reply RingSizer/Base/Extensions/Fonts.swift]: thanks, fixed it

File: RingSizer/Base/Extensions/UIFonts.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        return UIFont(name: FontFamily.SFProRounded.medium.name, size: size) ?? UIFont.systemFont(ofSize: size)
+    }
+
+    static func appBold(size: CGFloat) -> UIFont {
+        return UIFont(name: FontFamily.SFProRounded.bold.name, size: size) ?? UIFont.systemFont(ofSize: size)
+    }
+
+    static func appHeavy(size: CGFloat) -> UIFont {
+        return UIFont(name: FontFamily.SFProRounded.heavy.name, size: size) ?? UIFont.systemFont(ofSize: size)
+    }
```
Comment (vkostechko-applace) [inline RingSizer/Base/Extensions/UIFonts.swift:25]: it could be better. Attaching example 
PDFScanner_—_UIFont_Utils_swift_—_Edited

**Review by vkostechko-applace — CHANGES_REQUESTED** [review]

File: RingSizer/Base/Extensions/UIFonts.swift (hunk 3 above)
Comment (YuliaSimanavets) [reply RingSizer/Base/Extensions/UIFonts.swift:25]: how succinct :) corrected

**Review by vkostechko-applace — APPROVED** [review]

**Pull request merged**